### DISCUSSION TITLE
Customise i3xrocks status indicators

### DISCUSSION THREAD

**Discussion by carderne** [regolith-linux/regolith-desktop#670] [Q&A] [answered]
I want to make minor modifications to the default `i3xrocks` status indicators, mostly remove some and modify some beyond the scope of what Xresources allows.

Right now I'm deleting and editing files in `/usr/share/i3xrocks/conf.d/`, but that isn't resilient to upgrades.

Is there a better approach?

**Comment by kgilmer** [comment, marked as answer]
Hi @carderne , yes you can stage your versions of status indicators in your user directory.  Keep in mind it's either-or (vs summation) for them so you'll need to copy all of the config files into `~/.config/regolith2/i3xrocks/conf.d`.

See https://github.com/regolith-linux/regolith-i3-config/blob/master/partials/70_bar#L18
